.node-version

```diff
@@ -0,0 +1 @@
+v16.20.2
```

Comment:
I'd rather this target a major version rather than a specific one, but looking into it I'm not sure you can?

Reply:
I changed to v16 and it seems to work for me with fnm